classes/Cart.php

```diff
@@ -2158,6 +2158,10 @@ public function getGiftWrappingPrice($with_taxes = true, $id_address = null)
             return $wrapping_fees;
         }
 
+        if (!count($this->getProducts())) {
```

Comment:
IMO it should be on line 2152 (you could avoid getting a configuration key from database)